### DISCUSSION TITLE
I Love GENSHIN

### DISCUSSION THREAD

**Discussion by inderya** [share121/inter-knot#1255] [其他]
As the title said......
![Uploading 1672443653446e7e940ce4dd0f40a528c8d84f9a0e2874e28ead8df79c920562e993e182662d6.0.jpg…]()

**Comment by gaojiaobei** [comment]
I love Honkai:Starrail

**Comment by Panjk8** [comment]
me too

**Comment by shilijia-siweikongjian** [comment]
me, too!Genshin is such a good game！

**Comment by cpu644** [comment]
nuh uh

**Comment by tanls930** [comment]
i love honkai star rail
